install.sh

```diff
@@ -25,58 +26,80 @@ pip3 install pexpect
 pip3 install IKP3db
 pip install ikpdb
 
-echo "create folder structure"
+echo "###removing /etc/c9 if exists###"
+WORKING_DIR=/etc/c9
```

Comment:
You defined a WORKING_DIR, but no matter where I specified, this script still install it to /etc/c9. I think this will confuse users.